packages/naming.presets/test/origin/parse.test.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const assert = require('chai').assert;
+const naming = require('@bem/sdk.naming.entity')('origin');
+const parse = naming.parse;
+
+describe('origin parse', () => {
+    it('should not parse not valid string', () => {
+        const obj = parse('(*)_(*)');
+
+        assert.equal(obj, undefined);
+    });
+
+    it('should parse block', () => {
+        const obj = parse('block');
+
+        assert.equal(obj.block, 'block');
+    });
+
+    it('should parse mod of block', () => {
+        const obj = parse('block_mod_val');
+
+        assert.equal(obj.block, 'block');
+        assert.equal(obj.mod && obj.mod.name, 'mod');
```

Comment:
why do you need guard inside assert ? in tests you have absolute control

Reply:
It's a copy-paste